helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/algorithm/RebalanceAlgorithm.java

```diff
@@ -19,27 +19,21 @@
  * under the License.
  */
 
-import org.apache.helix.controller.rebalancer.waged.constraints.HardConstraint;
 import org.apache.helix.controller.rebalancer.waged.model.ClusterModel;
-import org.apache.helix.model.ResourceAssignment;
-
-import java.util.Map;
+import org.apache.helix.controller.rebalancer.waged.model.OptimalAssignment;
 
 /**
  * A generic rebalance algorithm interface for the WAGED rebalancer.
- *
- * @see <a href="Rebalance Algorithm">https://github.com/apache/helix/wiki/Design-Proposal---Weight-Aware-Globally-Even-Distribute-Rebalancer#rebalance-algorithm-adapter</a>
+ * @see <a
+ *      href="Rebalance Algorithm">https://github.com/apache/helix/wiki/Design-Proposal---Weight-Aware-Globally-Even-Distribute-Rebalancer#rebalance-algorithm-adapter</a>
```

Comment:
Can you use a `<pre> </pre>` tag to wrap the long url?
Would this look cleaner if we hide the link and show Rebalance Algorithm as a hyperlink?
```
<pre>
@see <a href="https://github.com/apache/helix/wiki/
Design-Proposal---Weight-Aware-Globally-Even-Distribute-Rebalancer
#rebalance-algorithm-adapter">Rebalance Algorithm</a>
</pre>
```

Reply:
Thanks! Good suggestion